elixir/phoenix-example/app/config/dev.exs

```diff
@@ -2,10 +2,11 @@ use Mix.Config
 
 # Configure your database
 config :appsignal_phoenix_example, AppsignalPhoenixExample.Repo,
-  username: System.get_env("POSTGRES_USER"),
-  password: System.get_env("POSTGRES_PASSWORD"),
-  database: System.get_env("POSTGRES_DB"),
-  hostname: "postgres",
+  username: System.get_env("POSTGRES_USER", "postgres"),
+  password: System.get_env("POSTGRES_PASSWORD", "postgres"),
```

Comment:
Are these all the default settings? Or can we add a script to create such as user and database?

Reply:
The user, password and port here are the default settings for a Postgres instance (or, at least, the ones for a default instance launched by Postgres.app) -- any value given to the database name will be used by `ecto.create` and `ecto.migrate`.